src/coffee.js

```diff
@@ -48,19 +50,14 @@ const coffee = [
   {
     name: "Beza",
     profile: "rich",
-    acidity: "sweet",
+    characteristic: "sweet",
     url: "https://impactroasters.dk/pages/about-coffe-coffee-guide",
     image: BezaImage,
-    characteristics: ["intense sweetness", "spicy", "forest berry aroma"],
+    taste: ["chocolate", "spicy", "floral"],
+    tastingProfile: ["intense sweetness", "spicy", "forest berry aroma"],
```

Comment:
Let's not reintroduce taste/tasting profile please. We only keep one, which is called `characteristics`.